data/ui/dialogs/main.ui

```diff
@@ -11,9 +11,9 @@
       <object class="HdyDeck" id="deck">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="transition_type">slide</property>
         <property name="can_swipe_back">True</property>
-        <property name="can_swipe_forward">True</property>
+        <signal name="notify::transition-running" handler="on_child_transition" swapped="no"/>
```

Comment:
swapped=no is the default, no need to specify it.

Reply:
Fixed.